webapp/src/main/webapp/templates/freemarker/edit/forms/autoCompleteDataPropForm.ftl

```diff
@@ -72,7 +72,7 @@ Also multiple types parameter set to true only if more than one type returned-->
     };
     var i18nStrings = {
         selectExisting: '${i18n().select_an_existing?js_string}',
```

Comment:
Do we need selectExisting field now? It looks as the new field is only needed? Also, if the selectExisting is needed, please rename it to selectAnExisting due to consistence.

Reply:
You are right, we don't need it. I removed it.